test/bdd/features/edv_e2e_api.feature

```diff
@@ -8,14 +8,18 @@
 @edv_rest
 Feature: Using EDV REST API
 
-  @e2e
+  @e2e_authorization
   Scenario: Full end-to-end flow. Create a data vault, store an encrypted document, and then retrieve the encrypted document. Query using an encrypted index. Update an encrypted document, and then retrieve the encrypted document.
     Given the wallet is registered as an OIDC client
     When the wallet redirects the user to authenticate at hub-auth
     And the user picks their third party OIDC provider
     And the user authenticates with the third party OIDC provider
     Then the user is redirected back to the wallet
     And the user has authenticated to the wallet
+    Then  Client sends request to create a new data vault and receives the vault location with access token
```

Comment:
Can you also add negative test? Failure in case of wrong access token.

Reply:
yes

Reply:
done